src/styles/breakpoints.ts

```diff
@@ -0,0 +1,39 @@
+import memoize from 'lodash/memoize';
+import { Dimensions } from 'react-native';
+
+const { width } = Dimensions.get('window');
+
+enum BREAKPOINTS {
+	// xsphone = 320,
+	smphone = 360, // small android phone: 360
+	// mdphone = 375, // iphoneSE and up
+	// lgphone = 390, // iphone15 and up
+	// xlphone = 400,
```

Comment:
nit: would drop 'phone' from these since it's redundant but other than LGTM
```suggestion
	// xs = 320,
	sm = 360, // small android phone: 360
	// md = 375, // iphoneSE and up
	// lg = 390, // iphone15 and up
	// xl = 400,
```